test/e2e/e2e_test.go

```diff
@@ -0,0 +1,141 @@
+package e2e
+
```

Comment:
Add license please. Same to other places.

Reply:
Added.

Reply:
Done.